src/Cake.Common/Tools/DotNetCore/DotNetCoreSettings.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Cake.Core.Tooling;
+
+namespace Cake.Common.Tools.DotNetCore
+{
+    /// <summary>
+    /// Contains common settings used by <see cref="DotNetCoreTool{TSettings}" />.
+    /// </summary>
+    public class DotNetCoreSettings : ToolSettings
```

Comment:
@patriksvensson A concreet implementation is needed for the executor. When you run `dotnet --help` you see that the verbosity property is part of all command. An alternative is an empty DotNetCoreExecutorSettings class.